grammar.js

```diff
@@ -208,8 +208,11 @@ module.exports = grammar({
         ")"
       )
     ),
+    
+    call_or_unqual_agg_body: $ => choice(seq("(", sep($._call_arg, ","), ")"),
```

Comment:
I think it would be better to separate these cases into different rules (so that we can tell the cases apart) and then make this an anonymous rule (as the cases are important rather than this rule).